benchmark/python/benchmark_hail/run/linalg_benchmarks.py

```diff
@@ -54,3 +54,10 @@ def blockmatrix_write_from_entry_expr_range_mt_standardize():
 def sum_table_of_ndarrays():
     ht = hl.utils.range_table(400).annotate(nd=hl.nd.ones((4096, 4096)))
     ht.aggregate(hl.agg.ndarray_sum(ht.nd))
+
+
+@benchmark()
+def to_matrix_table_row_major():
```

Comment:
can you prefix this with `block_matrix`? There's a method on Table with the same name, so we should be clear about what it's benchmarking.

Reply:
done